ALUM/ALUM/Views/SignUpPage/SignUpMenteeInfo.swift

```diff
@@ -0,0 +1,161 @@
+//
+//  SignUpMenteeInfo.swift
+//  ALUM
+//
+//  Created by Neelam Gurnani on 2/27/23.
+//
+
+import SwiftUI
+
+struct SignUpMenteeInfo: View {
+    
+    enum GradeType {
+        case nine
+        case ten
+        case eleven
+        case twelve
+    }
+    
+    @Environment(\.dismiss) var dismiss
+    @ObservedObject var viewModel: SignUpPageViewModel
+    @State var selectedGrade: GradeType? = nil
+    
+    var body: some View {
+        ZStack {
+            Color("ALUM White 2").edgesIgnoringSafeArea(.all)
+                .overlay(alignment: .top) {
+                    ProgressBarComponent(nodes: 3, filledNodes: 2, activeNode: 3)
+                        .frame(alignment: .top)
+                        .frame(maxWidth: .infinity)
+                        .padding()
+                        .background(Color.white.ignoresSafeArea(edges: .top))
+                        .frame(maxWidth: .infinity)
+                }
```

Comment:
action - same comment from SignUpJoinAs to change the structure of the Z, V, H stacks.